src/field_theory/intermediate_field.lean

```diff
@@ -352,6 +352,8 @@ S.to_subalgebra.range_val
 @[simp] lemma field_range_val : S.val.field_range = S :=
 set_like.ext' subtype.range_val
 
+instance : inhabited (S →ₐ[K] L) := ⟨S.val⟩
```

Comment:
What name does this instance get?

Reply:
`intermediate_field.alg_hom.nonempty`, which I think is fine?

Reply:
That would duplicate the namespace.
```suggestion
instance alg_hom.inhabited : inhabited (S →ₐ[K] L) := ⟨S.val⟩
```